utils/evaluate_utils.py

```diff
@@ -92,9 +93,48 @@ def class_PR_curves(
     return fig, PR_data
 
 
+def model_cm(
```

Comment:
can you rename this `model_confusion_matrix()` it is clearer than `cm`

Reply:
Yep! Completed in https://github.com/WayScience/phenotypic_profiling_model/pull/8 for simplicity.